src/components/table/examples/table-header-menu.tsx

```diff
@@ -0,0 +1,38 @@
+import { Component, h } from '@stencil/core';
+import { Column } from '../table.types';
+import { persons, Person } from './persons';
+
+/**
+ * Header menu
+ *
+ * @link persons.ts
+ */
+@Component({
+    tag: 'limel-example-table-header-menu',
+    styleUrl: 'table.scss',
+    shadow: true,
+})
+export class TableExampleHeadermenu {
```

Comment:
It would be great to display something when the menu icon is clicked in this example :slightly_smiling_face: